src/widgets/helper/ChannelView.hpp

```diff
@@ -179,6 +179,9 @@ class ChannelView final : public BaseWidget
 
     LimitedQueueSnapshot<MessageLayoutPtr> &getMessagesSnapshot();
 
+    // Returns true if message should be included
+    bool shouldIncludeMessage(const MessagePtr &m) const;
+
```

Comment:
When currently selected tab has a channel that is filtered by filters, and a message is received in the underlying channel, but the current tab does not show it (because it is filtered out by the filters), this needed to be used to make other tabs highlight on this message.

(It's not really needed, so it doesnt have to be public, since you can copy the code from inside with the use of public getFilterSet and similar)